lib/tests/validation/validation.test.js

```diff
@@ -458,6 +458,8 @@ describe("Gov Email domain validator", () => {
     ["test.hi@cds-snc.ca", true],
```

Comment:
I wonder if we should be moving any unit tests we come across to vitests?

Reply:
Will create a follow-up PR.